commands/settings/index.js

```diff
@@ -26,7 +26,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'pg',
   command: 'settings',
-  description: 'show current settings',
+  description: 'shows your database current postgres settings values',
```

Comment:
Comparing to other CLI addons, this should describe the action, rather than the command, so I believe this should stay `show`. And `values` is redundant here and it's pretty obvious that we're talking about `postgres`, so maybe something like:

> show your current database settings

?